examples/demo.py

```diff
@@ -3,8 +3,9 @@
 import numpy as np
 import pyopencl as cl
 
-a_np = np.random.rand(50000).astype(np.float32)
-b_np = np.random.rand(50000).astype(np.float32)
+rng = np.random.default_rng(seed=42)
+a_np = rng.random(50000, dtype=np.float32)
+b_np = rng.random(50000, dtype=np.float32)
```

Comment:
The demos optimize for line count more than reproducibility, so I'd prefer if those got reverted to their prior state.

Reply:
Backed out all the changes in the demos in https://github.com/inducer/pyopencl/compare/8811123351a421cefc917f6a247190aea56558e9..7ca6985d92ce2e8ac42a34752e68831420b19eaf.